src/__tests__/components/helpers/navBar.spec.js

```diff
@@ -15,110 +17,83 @@ describe('navBar', () => {
     wrapper = renderComponent();
   });
 
-  it('is a menu', () => {
-    expect(wrapper.type()).toEqual(Menu);
+  it('is an app bar', () => {
+    expect(wrapper.type()).toEqual(AppBar);
   });
 
-  describe('dropdown item', () => {
-    let dropdownItem;
+  describe('lefthand div', () => {
```

Comment:
These pieces are probably more than just divs. It would be helpful to describe what they really are. Is it a container? Does it display something useful? What does it do for us?